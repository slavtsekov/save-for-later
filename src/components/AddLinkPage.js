import React from 'react';

const AddLinkPage = () => (
    <div>
        Add Link page content.
    </div>
);

export { AddLinkPage as default };
