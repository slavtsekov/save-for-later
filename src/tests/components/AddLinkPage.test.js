import React from 'react';
import { shallow } from 'enzyme';
import AddLinkPage from '../../components/AddLinkPage';

test('should render AddLinkPage', () => {
    const wrapper = shallow(<AddLinkPage />);
    expect(wrapper).toMatchSnapshot();
});
