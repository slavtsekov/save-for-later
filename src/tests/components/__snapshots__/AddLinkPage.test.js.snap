// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AddLinkPage 1`] = `
<div>
  Add Link page content.
</div>
`;
